newIDE/app/src/MainFrame/EditorContainers/HomePage/TeamSection/TeamMemberProjectsView.js

```diff
@@ -0,0 +1,155 @@
+// @flow
```

Comment:
Could be interesting to make a story for this, so that it's easier in the future to work on these "sub pages". Though don't feel forced to do it now.